stoplight.js

```diff
@@ -0,0 +1,30 @@
+/*var redLight = $("#red");
+var redLight =button.on("click", function changeBackgroundColorToRed(){
+})
+
+var yellowLight = $("#yellow");
+var yellowLight = button.on("click", function changeBackgroundColorToYellow(){
+})
+
+var greenLight = $("#green");
+var greenLight = button.on("click", function changeBackgroundColorToGreen(){
+})*/
+
+$(function() {
+$('#red').click(function(){
+    $(this).css('background', 'red');
+
+  });
+});
```

Comment:
:+1: Your individual event listeners for changing the light color look good. You don't need to wrap it all in `($function(){ ... })` though.